### PR TITLE
Feature: Add italian dishes

### PR DESCRIPTION
**Pull request by AKincel18**
- [x] Add new folder with italian dishes in recipes.
- [x] Move pasta to italian dishes folder.
- [x] Add dishes to menu.

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -3,5 +3,5 @@ Cheesecake
 Orane juice
 Burger
 Fish and chips
-Pasta
 Pizza: margarita, capricciosa, hawaii
+Italian food: Bruschetta, Panzenella, Pasta
```
Comment (Niebieski18) [inline menu.txt:7]: The dishes are in alphabetical order, good.

File: recipes/Italian food/Bruschetta.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+An antipasto dish, bruschetta has grilled bread topped with veggies, rubbed garlic and tomato mix. A country bread sliced and topped with different toppings - the evergreen tomato-basil and an inventive mushroom-garlic. The classic Italian starter!
```
Comment (Niebieski18) [inline recipes/Italian food/Bruschetta.txt]: Could you provide some ingredients?

File: recipes/Italian food/Panzenella.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Panzenella is a Tuscan bread salad, ideal for summer. It does not follow a particular recipe, but the two ingredients that do not change are tomatoes and bread. This salad is great with a chilled glass of Prosecco and lots of sunshine!
```
Comment (Niebieski18) [inline recipes/Italian food/Panzenella.txt]: Could you provide some ingredients?

**Review by Niebieski18 — CHANGES_REQUESTED** [review]
Generally quite ok but required some correction.

**Comment by AKincel18** [comment]
Ok, thanks. Corrected in [b8d6984](https://github.com/AKincel18/git-tutorial/pull/1/commits/b8d698400610147b40d7c5fc8705f3a9302c2703) :)

**Review by Niebieski18 — APPROVED** [review]
Perfect.

**Pull request merged**